upgrading/blue_green_deployments.adoc

```diff
@@ -99,12 +99,12 @@ You must ensure that your current node hosts in production are labeled either
 $ oc get nodes
 ----
 
-. Ensure that all hosts have appropriate node labels. Consider that by default in
+. Ensure that all hosts have appropriate node labels. By default in
 {product-title} 3.9, all master hosts should now also be configured as
```

Comment:
@kalexand-rh be careful when you merge this. It seems to carry over content that is for 3.9 into older branches. 

Should also make sure that this is changed to {product-version}